includes/class-freemius.php

```diff
@@ -17425,7 +17472,13 @@ private function _get_latest_version_endpoint( $addon_id = false, $type = 'json'
 
             // If add-on and not yet activated, try to fetch based on server licensing.
             if ( is_bool( $is_premium ) ) {
-                $endpoint = add_query_arg( 'is_premium', json_encode( $is_premium ), $endpoint );
+                $endpoint = add_query_arg(
+                    array(
+                        'is_premium' => json_encode( $is_premium ),
+                        'readme'     => json_encode( $is_premium )

```

Comment:
@fajardoleo did you verify that we actually need the readme on every request when running from a premium context? The readme adds quite a lot of content to the result (and bandwidth to our servers).

For example, when `_fetch_newer_version()` executed and there's no newer version, we don't really need to return the readme. Maybe to avoid duplicate calls (once to check if there's a newer version and then another to get the readme), we can enrich the endpoint with a `newer_than` param which will receive the current's product version and will only return a result when the latest matching version actually newer than the one specified by `newer_than`? (behave like that only if that param is specified